src/redis-cli.c

```diff
@@ -1532,6 +1536,24 @@ static int parseOptions(int argc, char **argv) {
             i = j;
         } else if (!strcmp(argv[i],"--cluster") && lastarg) {
             usage();
+        } else if ((!strcmp(argv[i],"--cluster-only-masters") && !lastarg)) {
+            char *cmd = argv[++i];
+            int j = i;
+            while (j < argc && argv[j][0] != '-') j++;
+            if (j > i) j--;
+            createClusterManagerCommand(cmd, j - i, argv + i + 1);
```

Comment:
See previous comment. You cannot create the command following the `--cluster-only-*` options.
Please take a look at other --cluster commands and follow the same syntax pattern.

Reply:
done